packages/circuit-ui/components/Button/Button.tsx

```diff
@@ -82,6 +87,34 @@ export type ButtonProps = BaseProps & LinkElProps & ButtonElProps;
 
 const BORDER_WIDTH = '1px';
 
+const COLOR_MAP = {
+  default: {
+    default: 'p500',
+    hover: 'p700',
+    active: 'p900',
+  },
+  destructive: {
+    default: 'danger',
+    hover: 'r700',
+    active: 'r900',
```

Comment:
Aren't we creating a dependency here between the semantic color and the `r***` ones? For example if we change the hue of `danger` it won't be reflected in the other states. Maybe we could use a brightness filter instead? Or extend the semantic colors? What do you think?

Reply:
You make a valid point. Unfortunately, we're in this in-between phase where the colors haven't been fully converted into semantic colors yet. In fact, these shades of red were removed from the palette 🤔 I've added a comment in Figma to ask Giao-Chung about it. 

I believe a brightness filter would be too imprecise.

Reply:
I agree that brightness isn't the best solution - let's keep this for now if we're revisiting colors to make them semantic across the board 👍

Reply:
I found out from Giao-Chung that these colors do in fact exist in the new color palette albeit under different names. I agree that we should revisit this with the semantic design token task 👍🏻